user/token.go

```diff
@@ -69,9 +71,14 @@ func CreateSessionToken(data *TokenData, config TokenConfig) (*SessionToken, err
 	token.Claims["usr"] = data.UserId
 	token.Claims["dur"] = data.DurationSecs
 	token.Claims["exp"] = expiresAt
+	token.Claims["iss"] = "tidepool/shoreline"
```

Comment:
I think that it might be nice for `iss` to be the issuing URL (eg https://api.tidepool.org/auth/login). This would enable us to differentiate tokens in different environments.

Reply:
Good idea.